test/test_pretty/test_pretty.ml

```diff
@@ -1 +1 @@
-let tests = List.flatten [ Test_color.tests ]
+let tests = List.flatten []
```

Comment:
I was testing only my own Color implementation but now I don't have tests anymore, lol.

They'll be added later, don't worry. There're lots of tests to write.